config/slim_lint.yml

```
@@ -26,11 +25,11 @@ linters:
       - Layout/MultilineOperationIndentation
       - Layout/ParameterAlignment
       - Layout/TrailingEmptyLines
-      - Lint/Void
-      - Metrics/BlockLength
+      - Lint/Void  #除外検討したが見送り
```

このタイプのコメントがこの設定ファイル中で本当に機能するのかが気になりました。

また、コメントはだいたい`#`から1スペース開いてることが多いのでそれに合わせた方がいいかなと思いました。

```suggestion
      - Lint/Void  # 除外検討したが見送り
```

---

@komagata 
`.yml`の形式のファイルのコメントに仕様について調べてみました。
いくつかサイトを見たところ、コメントの前に`#`を付ける形でよいということでした。
ということで、コメントのタイプはそのままで据え置き、`#`の後に1スペース追加のみしております。
確認よろしくお願いいたしますm(__)m偽t

以下確認したサイト例です
https://www.redhat.com/ja/topics/automation/what-is-yaml
https://apidog.com/jp/blog/how-to-write-a-yaml-file/